winter-website.css

```diff
@@ -0,0 +1,4 @@
+body {
+	color: #e8f3fc;
+	background-color: black;
```

Comment:
should be consistent with color types so you should continue to use hex here. the code for black is #000000

Reply:
done